var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -47,29 +48,21 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
         "~hiop~ipopt @:1.4",
         msg="ExaGO needs at least one solver enabled. PFLOW only mode is supported in 1.5+",
     )
-    # We will better support minimal builds with Python in future.
     # You can use Python with PFLOW if desired ~ipopt~hiop
     conflicts(
-        "~hiop~ipopt+python", msg="ExaGO Python wrapper requires at least one solver enabled."
+        "~hiop~ipopt+python @:1.5.0",
+        msg="ExaGO Python wrapper requires at least one solver enabled.",
     )
 
     # Dependencies
+    depends_on("python@3.6:", when="@1.3.0:+python")
     depends_on("py-pytest", type=("build", "run"), when="@1.5.0:+python")
     depends_on("py-mpi4py", when="@1.3.0:+mpi+python")
     depends_on("pkgconfig", type="build")
     depends_on("mpi", when="+mpi")
     depends_on("blas")
     depends_on("ipopt~mumps", when="+ipopt")
     depends_on("cuda", when="+cuda")
-    depends_on("raja", when="+raja")
-    depends_on("umpire", when="+raja")
-
-    depends_on("umpire@6.0.0", when="@1.1.0: +raja")
-    depends_on("raja@0.14.0", when="@1.1.0: +raja")
-    depends_on("camp@0.2.3", when="@1.1.0: +raja")
```

Comment:
I would keep this in ExaGO configuration instead of letting HiOp set RAJA and related dependencies.